src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/ChimericAlignment.java

```diff
@@ -294,6 +294,24 @@ public boolean isNotSimpleTranslocation() {
                                         involvesRefPositionSwitch(regionWithLowerCoordOnContig, regionWithHigherCoordOnContig));
     }
 
+    /**
+     * See {@link #isLikelyInvertedDuplication()}
+     */
+    @VisibleForTesting
+    public static boolean isLikelyInvertedDuplication(final AlignmentInterval one, final AlignmentInterval two) {
```

Comment:
Now that this method is on `ChimericAlignment` could you make it non static and remove the parameters (since they're generally the instance variables `regionWithLowerCoordOnContig` and `regionWithHigherCoordOnContig`? Or at least you could create a non-static version that defers to the static one to be used when applicable?

Reply:
Yes, that is available several lines down. And the same case with `isLikelySimpleTranslocation()`: one static one for easier testing, and one non-static one for easier usage.

Reply:
Oh, sorry I missed that. Carry on then.